src/TLuaInterpreter.cpp

```diff
@@ -2190,19 +2190,21 @@ int TLuaInterpreter::disableTimer( lua_State *L )
 
 int TLuaInterpreter::enableKey( lua_State *L )
 {
-    string luaSendText="";
+    string luaNameID = "";
     if( ! lua_isstring( L, 1 ) )
     {
-        lua_pushstring( L, "enableKey: wrong argument type" );
+        lua_pushstring( L, tr( "enableKey: bad argument #1 type (key name as string expected, got %1!)" )
+            .arg( luaL_typename( L, 1 ) )
+            .toUtf8().constData() );
         lua_error( L );
         return 1;
     }
     else
     {
-        luaSendText = lua_tostring( L, 1 );
+        luaNameID = lua_tostring( L, 1 );
     }
     Host& host = getHostFromLua(L);
-    QString text(luaSendText.c_str());
+    QString text(luaNameID.c_str());
```

Comment:
This line is now redundant as we already have it as a `QString`, and the following line uses `keyName` instead of `text`! 🤓 

This sort of thing also should be done in the remaining changed `TLuaInterpreter` functions...